public/js/showoff.js

```diff
@@ -1693,6 +1693,10 @@ function keyDown(event){
   if (event.ctrlKey || event.altKey || event.metaKey) {
     return true;
   }
+  // in case the user is typing in text fields
+  if (document.activeElement.type.substring(0,4) === "text") {
```

Comment:
`document.activeElement` is supported by the major browsers, but is still technically "experimental."

https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/activeElement#Browser_compatibility

I'm okay keeping it, but an alternative is putting this earlier in the code:

    $('input, textarea').focus(function() {
      // disable hotkeys
    });